VMBackup/main/fsfreezer.py

```diff
@@ -248,10 +238,23 @@ def freeze_safe(self,timeout):
             self.logger.log(error_msg, True, 'Error')
         return freeze_result,timedout
 
+    def releaseFileLock(self):
+        if (self.isAquireLockSucceeded == True):
+            try:
+                fcntl.lockf(self.file, fcntl.LOCK_UN)
+                self.file.close()
+            except:
+                pass
```

Comment:
even though you intend to eat the exception, better log the stack trace and pass the exception.
it would be helpful in debugging if there are any errors

Reply:
added logs